integration-tests/build.gradle.kts

```diff
@@ -28,6 +28,9 @@ val GENERATION_TESTS = listOf(
     GenerationTest("generateBuildSolutionParallel", "generate-build-solution",
         listOf("--model", "my.build.script", "--parallel-generation-threads=4"),
         expectation = GenerationTestExpectation.SuccessWithFiles("solutions/my.build/build.xml")),
+    GenerationTest("generateBuildSolutionNonStrict", "generate-build-solution",
+        listOf("--model", "my.build.script", "--disable-strict-mode"),
```

Comment:
Let's skip this test.

We should later change the tests for generation settings to just test the settings without running MPS. The more integration tests we have, the slower they run.

Reply:
Removed